runners/direct-java/src/main/java/org/apache/beam/runners/direct/ExecutorServiceParallelExecutor.java

```diff
@@ -453,7 +453,9 @@ private void fireTimers() throws Exception {
               }
               KeyedWorkItem<?, Object> work =
                   KeyedWorkItems.timersWorkItem(keyTimers.getKey().getKey(), delivery);
-              LOG.warn("Delivering {} timers for {}", delivery.size(), keyTimers.getKey().getKey());
+              LOG.debug("Delivering {} timers for {}",
```

Comment:
is this worth wrapping inside of a `if (LOG.isDebugEnabled())`? aka how expensive are the `size`/`getKeygetKey` vs the frequency of call?